src/requirements/data/majors/comm.ts

```diff
@@ -1,33 +1,26 @@
 import { CollegeOrMajorRequirement } from '../../types';
-import { includesWithSingleRequirement, includesWithSubRequirements } from '../checkers-common';
+import { includesWithSingleRequirement } from '../checkers-common';
 
 const commRequirements: readonly CollegeOrMajorRequirement[] = [
   {
     name: 'Core Courses',
     description: '15 credits in COMM 1101, COMM 1300, COMM 2010, COMM 2310, COMM 2820',
     source: 'https://communication.cals.cornell.edu/undergraduate-program/major-requirements/',
-    checker: includesWithSubRequirements(
-      ['COMM 1101'],
-      ['COMM 1300'],
-      ['COMM 2010'],
-      ['COMM 2310'],
-      ['COMM 2820']
+    checker: includesWithSingleRequirement(
+      'COMM 1101',
+      'COMM 1300',
+      'COMM 2010',
+      'COMM 2310',
+      'COMM 2820'
     ),
-    subRequirementProgress: 'every-course-needed',
     fulfilledBy: 'credits',
     minCount: 15,
   },
   {
     name: 'Focus Area',
     description: '6 credits in COMM 2200, COMM 2450, COMM 2760, or COMM 2850',
     source: 'https://communication.cals.cornell.edu/undergraduate-program/major-requirements/',
-    checker: includesWithSubRequirements(
-      ['COMM 2200'],
-      ['COMM 2450'],
-      ['COMM 2760'],
-      ['COMM 2850']
-    ),
-    subRequirementProgress: 'any-can-count',
+    checker: includesWithSingleRequirement('COMM 2200', 'COMM 2450', 'COMM 2760', 'COMM 2850'),
     fulfilledBy: 'credits',
     minCount: 6,
   },
```

Comment:
Similarly, can we do `fulfilledBy: 'courses'`? I can see why we chose credits but just wanted to hear your thoughts.